test/BaselineOutput/Common/Text/bag_of_words.tsv

```diff
@@ -0,0 +1,12 @@
yet	39851:CD	39884:up,	39892:note	39963:well-covered	39967:uses.	39972:successful	39987:(on	39989:erosion.	39990:refering	40024:any	40030:fight	40056:manifested	40078:indicates	40101:Ryan	40107:conversation	40109:Better	40120:Cookies	40138:is,	40225:record	40252:GETTING	40256:DOES	40289:MY	40316:pointing	40336:fact	40341:ill	40348:{{helpme}}	40349:guidelines.	40350:Articles	40352:Tempe,	40368:settled	40383:acts	40391:is	40441:training	40446:protected	40456:continents	40500:giver's	40502:{below,violation}	40508:time?	40510:Russian?	40527:google	40569:agrees	40614:not.	40636:interested	40667:administrators,	40672:SEAHORSE	40693:Librier	40696:basic	40734:are.	40738:{watch,Outfitters.}	40745:Invincible	40761:placed	40769:orogenesis.	40771:now	40777:state	40787:excited,	40817:pattern	40841:grocer	40846:"question:"	40929:date.	40970:YOUR	40972:take	40976:pedant.	40988:RIDICULOUS	41015:life	41030:stands	41053:statements	41060:ChrisO's	41101:RAGIB	41119:fit	41155:I'd	41164:ahistorical	41205:Reeves'	41215:boyfriend	41233:wikipedia.org.	41243:Base64	41254:usage,	41259:206.45.24.242	41264:has.	41268:Need	41269:":::The"	41278:"""war."""	41289:speculations.	41290:outspoken	41310:comments==	41315:basements	41337:wikipedia.org	41369:tyrant	41371:DRINK!	41373:Daddy-	41379:Euro	41380:respected	41381:UNLEASH	41428:fans	41441:denotes	41469:==Reply	41475:ANYTHING	41488:THAT'S	41508:demolish	41546:specific	41554:"""sounds"	41583:finish	41588:Slim	41598:WILL	41632:provide	41645:calls	41651:All	41696:YOU.	41709:{ignoramus,album.}	41742:Labelled	41753:absolutely	41767:languages	41769:"05:18"	41770:lunch	41771:ME...	41772:(among	41801:continue	41824:{providinga,Project}	41831:adult	41883:gives	41890:denials.	41895:would	41916:wiki	41918:netral	41942:France,	41971:Big	41994:bring	42017:perspective.	42023:seriously,	42044:"denialism"""	42046:job,	42048:Filed	42059:crime	42077:INSANE.	42080:{it!,angry}	42105:words,	42106:accessible	42110:#6	42156:{disambiguation,generals.}	42180:deleate	42235:treat	42254:posts	42257:god,	42261:repeatedly	42279:already!	42311:hmm...	42313:you!	42364:english	42384:(at	42391:(football	42406:sales	42408:taking	42455:{legitimate,what?}	42484:gets	42485:there.	42513:ideas	42518:""">100"	42521:GAYTOURAGE...	42536:denialists	42563:official	42572:lover	42588:"::::::What"	42614:git),	42708:detail	42717:AS	42766:days	42768:5	42844:views	42856:Rhino	42897:/	42916:Saman	42935:Good	42955:suit	42979:BNP	42989:EXPECTO	42997:busted.	43004:questions,	43018:August	43055:ARTICLES,	43086:ONLY	43131:WoohooKitty,	43146:AWFUL	43165:evidence?	43180:ISP	43181:what's	43184:talkpages	43187:judgment,	43188:discredited	43238:List	43269:{endanger,directors.}	43272:{copy,AIDs,}	43276:{denailism,,EACH}	43305:since	43308:trolling,	43324:manner,	43330:suicides	43345:email	43356:Reeves	43366:nonsense.	43414:TYPHOID	43415:custody	43422:homepage.	43462:replication	43470:Pathetic	43505:are,	43538:doubt	43551:PITY	43559:LEAVE	43597:perspective?	43604:me?	43632:motivation,	43635:o
```

Comment:
>==RUDE==	Dude,	you	are	rude	upload	th [](start = 5, length = 37)

I am curious about this line in the output.  

Looks like its a concat of the 4 rows from the dataset.

Reply:
Well I believe it should be (almost) concat of all rows from dataset.
This is a header, and we put slotnames into header for VBuffer alike columns.
So this is basically a dictionary of all words (from word bag) plus hashes for all words from wordbaghash.

---
In reply to: [234732025](https://github.com/dotnet/machinelearning/pull/1670#discussion_r234732025) [](ancestors = 234732025)

Reply:
cool. thanks for the explanations

---
In reply to: [234733325](https://github.com/dotnet/machinelearning/pull/1670#discussion_r234733325) [](ancestors = 234733325,234732025)